cli/args.go

```diff
@@ -82,7 +82,7 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 
 	opts.TerraformPath = filepath.ToSlash(terraformPath)
 	opts.AutoInit = !parseBooleanArg(args, OPT_TERRAGRUNT_NO_AUTO_INIT, os.Getenv("TERRAGRUNT_AUTO_INIT") == "false")
-	opts.NonInteractive = parseBooleanArg(args, OPT_NON_INTERACTIVE, false)
+	opts.NonInteractive = parseBooleanArg(args, OPT_NON_INTERACTIVE, os.Getenv("TF_INPUT") == "false" || os.Getenv("TF_INPUT") == 0)
```

Comment:
One more thing: `Getenv` returns a `string`, so it would have to be `== "0"`.

Reply:
@brikis98  Thanks, this is literally the first time I have ever touched golang. I appreciate all the help I have received in the issues and want to be net positive - hope #370 helps too.